src/gpu/gpu_utils.hpp

```diff
@@ -21,9 +21,9 @@
 #include <map>
 #include <vector>
 
-#include "oneapi/dnnl/dnnl.h"
-
+#include "common/convolution_pd.hpp"
```

Comment:
Doesn't feel right that utils now depend on convolution. It should probably leave in gpu_reorder.hpp or maybe even in a new standalone hpp + cpp pair, preferably a cpp file owning this dependency alone.